### PR TITLE
[8.1] fix updating folder sizes with cross storage move

### PR DESCRIPTION
**Pull request by icewind1991**
Backport of #25101 to 8.1

cc @owncloud/filesystem

### CODE REVIEW

**Comment by PVince81** [comment]
Tested, works :+1: 
Thanks!

Second review ? @rullzer @nickvergessen @mmattel

File: lib/private/files/cache/updater.php [hunk 1]
```diff
@@ -170,6 +170,7 @@ public function rename($source, $target) {
 		list($targetStorage, $targetInternalPath) = $this->view->resolvePath($target);
 
 		if ($sourceStorage && $targetStorage) {
+			$sourceCache = $sourceStorage->getCache($sourceInternalPath);
 			$targetCache = $targetStorage->getCache($sourceInternalPath);
```
Comment (nickvergessen) [inline lib/private/files/cache/updater.php]: `$targetInternalPath` here?

Comment (icewind1991) [inline lib/private/files/cache/updater.php]: fixed

**Comment by nickvergessen** [comment]
works :+1:

**Pull request merged**